### PR TITLE
Adds format for webpage content

### PR DESCRIPTION
**Pull request by guavafanclub**

### CODE REVIEW

File: portfolio/src/main/webapp/projects.html [hunk 1]
```diff
@@ -20,35 +20,93 @@ <h2><a href="work-history.html">Work History</a> |
...
+            <li>Geo-tag was approved and deployed by Snapchat</li>
+          </ul>
+        </div>
+        <div class="spread-content">
+            <img src="images/tepper.jpeg" alt="Carnegie Mellon Building"/>
+        </div>
+      </div>
     </div>
   </body>
 </html>
```
Comment (pruess-google) [inline portfolio/src/main/webapp/projects.html:112]: Please add a new line to the end of this file

File: portfolio/src/main/webapp/work-history.html [hunk 2]
```diff
@@ -19,41 +19,69 @@ <h2><a href="work-history.html">Work History</a> |
...
+          </i></h4>
+          <p><i>Fall 2018 - March 2020</i></p>
+          <ul>
+            <li>Improve 6th-9th grade public school students’ Keystone exam scores by tutoring them weekly.</li>
+          </ul>
+        </div>
+      </div>
     </div>
   </body>
 </html>
```
Comment (pruess-google) [inline portfolio/src/main/webapp/work-history.html:87]: please add a new line to the end of this file

Comment (alekseychaikovsky) [inline portfolio/src/main/webapp/work-history.html:87]: newline 🙀

**Comment by guavafanclub** [comment]
> Everything looks good. Does this change affect the appearance of your website, or is it only reformatting the code?
> 
> If it does affect the appearance, would you mind including some before and after screenshots so we can see the difference?

Of course.
Before: https://screenshot.googleplex.com/KpTWASCTnMX.png
After: https://screenshot.googleplex.com/6OGga99N38o.png
@alekseychaikovsky (@ing you on this in case you want to see as well)

**Pull request merged**